arbitrum-docs/node-running/building-nitro-locally.mdx

```diff
@@ -0,0 +1,50 @@
+# Building nitro locally
+
+### 0. Setup Prerequisites
+  ```bash
+  apt install git curl build-essential cmake npm golang clang make
```

Comment:
It might be worth noting what debian version this guide was made for

Reply:
done.